packages/components/icon/src/icon.tsx

```diff
@@ -62,6 +63,8 @@ export const Icon = forwardRef<IconProps, "svg">((props, ref) => {
       <ui.svg
         as={element}
         className={cx("ui-icon", className)}
+        aria-hidden={ariaHidden}
```

Comment:
Change `aria-hidden={ariaHidden}` to `aria-hidden`.

Reply:
fixed
2f10ba6